codegen/codegen_test.go

```diff
@@ -76,7 +76,7 @@ func TestCodeGen(t *testing.T) {
 		}
 		`,
 		func(t *testing.T, cg *CodeGen) solver.Request {
-			id, err := cg.LocalID(".")
+			id, err := cg.LocalID(context.Background(), ".")
```

Comment:
Can we deduplicate this by making a `ctx := context.Background()` at the top?

Reply:
fixed